src/webWorker/decodeAndTransformTask.js

```diff
@@ -60,7 +60,8 @@ function _handler (data, doneCallback) {
         )
       }).catch(
         (error) => {
-          throw new Error(`Failed to transform frame: ${error}`)
+          console.warn('Unable to transform frame', error)
```

Comment:
Failing to load hte icc profiles shouldn't prevent display - this was happening sometimes depending on the load path.  I've mostly fixed this elsewhere, but I'm not 100% sure that all the paths are fixed.